.goreleaser.yml

```diff
@@ -54,19 +56,6 @@ changelog:
 
 archives:
   - name_template: '{{ .ProjectName }}_{{ title .Os }}_{{ .Arch }}'
-    format_overrides:
-      - goos: windows
-        format: zip
-
-brews:
-  - repository:
-      owner: toshimaru
-      name: homebrew-nyan
-    description: Colored cat command which supports syntax highlighting
-    homepage: https://github.com/toshimaru/nyan
-    license: MIT
-    test: |
-      system "#{bin}/nyan", "-v"
```

Comment:
Add `homebrew_casks` section to yaml.

ref. https://github.com/goreleaser/goreleaser/blob/735ed6f3c5d3db8de1d840de4ea5f8b5f94ef884/.goreleaser.yaml#L221C1-L237